src/workerd/server/workerd-api.c++

```diff
@@ -446,7 +533,18 @@ void WorkerdApi::compileModules(
     if (hasPythonModules(confModules)) {
       KJ_REQUIRE(featureFlags.getPythonWorkers(),
           "The python_workers compatibility flag is required to use Python.");
-      // Inject pyodide bootstrap module.
+      // Inject Pyodide bundle
+      if(util::Autogate::isEnabled(util::AutogateKey::PYODIDE_LOAD_EXTERNAL)) {
+          if (fetchPyodideBundle(impl->pythonConfig, "dev"_kj)) {
+            modules->addBuiltinBundle(getPyodideBundle("dev"_kj), kj::none);
+          } else {
+            auto version = KJ_ASSERT_NONNULL(getPythonSnapshotRelease(featureFlags)).getPyodide();
```

Comment:
I wonder if we actually want to use more than just the Pyodide version in the bundle filename. Presumably we want `pyodide` + `pyodideRevision` + `backport`. @hoodmane what are your thoughts on this?

Reply:
Agreed, `pyodide` + `pyodideRevision` + `backport` sounds right. But also, we need to upload a bundle with this name before this will work...

Reply:
I got it!